### PR TITLE
feat: #82 new post on study days

### PR DESCRIPTION
**Pull request by bms63**
Quick demo of `derive_vars_dy()` function.  I wanted to put something technical out there with Stefan's post.

### CODE REVIEW

File: posts/2023-08-08_study_day/study_day.qmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  "004", "2014-03-17T23:59:59", "2014-03-19T13:09:O9", "2014-05-04"
+) %>%
+  mutate(
+    TRTSDTM = as_datetime(TRTSDTM),
+    ASTDTM = as_datetime(ASTDTM),
+    AENDT = ymd(AENDT)
+  )
+```
+
+Okay! Next we run our dataset through `derive_vars_dy()`, specifying the `reference_date` of the variable you want the calculation to be done against and the `source_vars`argument with the variables to help calculate the relative days.
```
Comment (manciniedoardo) [inline posts/2023-08-08_study_day/study_day.qmd:48]: This sentence was slightly confusing at a first glance, perhaps change to a bullet-pointed list. Something like

Okay! Next we run our dataset through `derive_vars_dy()`, specifying:
- the reference date of the variable we want the calculation to be done against (`reference_date` argument);
- the variables required to help calculate the relative days  (`source_vars` argument).

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Great post @bms63 - short and sweet. I just identified one sentence that was slightly confusing at first blush and proposed an amendment.

File: R/create_blogpost.R
Comment (StefanThoma) [inline R/create_blogpost.R:1]: Not sure if this needs to be committed.

File: posts/2023-08-08_study_day/study_day.qmd [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Let's build some dummy data with 4 subjects, a start date/time for treatment (`TRTSDTM`), an analysis start date/time variable (`ASTDTM`) and an analysis end date variable (`AENDT`).
+
+```{r, message = FALSE}
+library(admiral)
+library(lubridate)
+library(dplyr)
+
+adam <- tribble(
+  ~USUBJID, ~TRTSDTM, ~ASTDTM, ~AENDT,
```
Comment (StefanThoma) [inline posts/2023-08-08_study_day/study_day.qmd:35]: I think we should align the commas of the columns.

**Review by StefanThoma — APPROVED** [review]

File: posts/2023-08-08_study_day/study_day.qmd (hunk 2 above)
Comment (bms63) [reply posts/2023-08-08_study_day/study_day.qmd:35]: styler forced this non-alignment. Just going to leave as is.

File: R/create_blogpost.R
Comment (bms63) [reply R/create_blogpost.R:1]: ah whoops.  just going to leave as is :)

**Pull request merged**